_data/simple-icons.json

```diff
@@ -3639,12 +3640,12 @@
         {
             "title": "Google Hangouts",
             "hex": "0C9D58",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/e/ee/Hangouts_icon.svg"
+            "source": "https://about.google/brand-resource-center/logos-list/"
```

Comment:
I'm not sure this is a good source replacement since the logo is in PNG in this page.

Reply:
changed source to commons wiki svg